packages/react/src/drafts/IssueLabel/IssueLabel.tsx

```diff
@@ -43,8 +45,22 @@ export interface IssueLabelProps extends React.PropsWithChildren {
 }
 
 export function IssueLabel({children, fillColor, size = 'small', variant = 'gray', ...rest}: IssueLabelProps) {
+  const {resolvedColorScheme} = useTheme()
+  const mode = resolvedColorScheme?.startsWith('dark') ? 'dark' : 'light'
+  // I define those bgColors as we can't easily get them from primitives,
```

Comment:
Would it be possible to get these values from primitives in the future? Would be awesome to be able to interact with it as a package to get the values 👀